packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -15,6 +15,8 @@ export default class QuanticTab extends LightningElement {
   @api expression;
   /** @type {string} */
   @api engineId;
+  /** @type {boolean} */
+  @api initAsActive = false;
```

Comment:
Why not use something like this?

```js
  @track _isActive = false;
  @api set isActive(active) {
    this._isActive = active;
  }

...

  initialState: {
    isActive: this._isActive
  }
```

Reply:
Because I didn't think of this but I love it!

Reply:
... have you actually ever made this work?